source/_integrations/yolink.markdown

```diff
@@ -40,6 +40,13 @@ Home Assistant will use account linking provided by Nabu Casa for authenticating
 If you want to use separate credentials, please contact <service@yosmart.com> to obtain a `client_id` and `client_secret`. Then you can add your credentials via application credentials. Settings > Devices & Services > click the menu (three dots at the top right of the screen) and then **Application Credentials**. Enter your credentials in the pop-up window.
 {% enddetails %}
 
+{% warning %}
```

Comment:
Thank you for the comments, @matrixd2 :+1: 
I think this could be placed in a section at the bottom of the page, called ## Community notes.
I would make a suggestion, but it is too far outside for me to let me add it.